server/src/index.js

```diff
@@ -1,67 +1,72 @@
 const express = require('express')
-const fs = require('fs')
 const path = require('path')
+const http = require('http')
+const fs = require('fs')
 
 const { ApolloVoyagerServer } = require('@aerogear/apollo-voyager-server')
+const { KeycloakSecurityService } = require('@aerogear/apollo-voyager-keycloak')
+const metrics = require('@aerogear/apollo-voyager-metrics')
+const auditLogger = require('@aerogear/apollo-voyager-audit')
 
+const config = require('./config/config')
 const connect = require('./db')
-const schema = require('./schema')
-const http = require('http');
+const { typeDefs, resolvers } = require('./schema')
 
-const dbOptions = {
-  database: process.env.DB_NAME || 'users',
-  user: process.env.DB_USERNAME || 'postgresql',
-  password: process.env.DB_PASSWORD || 'postgres',
-  host: process.env.DB_HOSTNAME || '127.0.0.1',
-  port: process.env.DB_PORT || '5432'
-}
+let keycloakService = null
 
-const PORT = 4000
+// if a keycloak config is present we create
+// a keycloak service which will be passed into
+// ApolloVoyagerServer
+if (config.keycloakConfig) {
+  keycloakService = new KeycloakSecurityService(config.keycloakConfig)
+}
 
 async function start() {
+  
   const app = express()
+  const httpServer = http.createServer(app)
```

Comment:
we probably need to enable CORS here to allow the local client app to work with a remote backend server.